src/core/annotation.js

```diff
@@ -941,6 +941,56 @@ class Annotation {
     return null;
   }
 
+  get hasSomeText() {
```

Comment:
For consistency, with the new property name:
```suggestion
  get hasTextContent() {
```